ci/scrub-kicad-project-files

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# prototype-kicad-library-ci
+
+# File: /ci/scrub-kicad-project-files
+# Description: prototype-kicad-library-ci repository KiCad project file scrubbing script.
```

Comment:
Please change `repository KiCad project file scrubbing script` to `repository CI KiCad project files scrubbing script` for consistency with other CI scripts. This change will push the line beyond the 90 column limit (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide) and thus the description should span two lines.
```suggestion
# Description: prototype-kicad-library-ci repository CI KiCad project files scrubbing
#       script.
```